### PR TITLE
Fix/web subtree

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.controller.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+angular.module('controllers').controller(
+  'controllers.Domain.Options.Delete',
+  class DomainDnsLockCtrl {
+    constructor($rootScope, $scope, $translate, Alerter, Domain) {
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.controller.js]:
```suggestion
    /* @ngInject */
    constructor($rootScope, $scope, $translate, Alerter, Domain) {
```

File: packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      data-loading="ctrl.loading"
+      data-on-dismiss="ctrl.closeModal()"
+      data-primary-action="ctrl.deleteDomain()"
+      data-primary-disabled="ctrl.loading"
+      data-primary-label="{{:: 'domain_tab_options_delete_confirm_button' | translate }}"
+      data-secondary-action="ctrl.closeModal()"
+      data-secondary-disabled="ctrl.loading"
+      data-secondary-label="{{:: 'domain_tab_options_delete_cancel' | translate }}">
+        <p data-translate="domain_tab_options_delete_confirm"></p>
+        <oui-message type="warning">
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.html]:
```suggestion
        <oui-message data-type="warning">
```

File: packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.controller.js (hunk 1 above)
Comment (cbourgois) [reply packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.controller.js]: Not part of this PR. It's just a `git subtree pull` from the original `develop` branch (https://github.com/ovh-ux/ovh-manager-web).

File: packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.html (hunk 2 above)
Comment (cbourgois) [reply packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.html]: Not part of this PR. It's just a `git subtree pull` from the original `develop` branch (https://github.com/ovh-ux/ovh-manager-web).

**Pull request merged**